backend/.dockerignore

```diff
@@ -0,0 +1,6 @@
+.git
+.dockerignore
+docker-compose*.yml
+npm-debug.log*
+Dockerfile
+node_modules
```

Comment:
please add an empty line at the end of the file
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file